doc/release-notes.md

```diff
@@ -274,6 +277,17 @@ To enable block pruning set `prune=<N>` on the command line or in
 `bitcoin.conf`, where `N` is the number of MiB to allot for
 raw block & undo data.
 
+A value of 0 disables pruning. The minimal value above 0 is 550. Your
+wallet is as secure with high values as it is with low ones. Higher
+values merely ensure that your node will not shut down upon blockchain
+reorganizations of more than 2 days - which are unlikely to happen in
+practice unless there is a hard fork. In future releases, a higher value
```

Comment:
I would strike "unless there is a hard fork" from this sentence; hard forks shouldn't have much bearing on the likelihood of a 288 block reorg.